Dockerfile

```diff
@@ -1,10 +1,16 @@
-FROM alpine:3.2
+FROM ruby:2.3-alpine
 
 MAINTAINER PRX <sysadmin@prx.org>
 
-RUN apk update && apk --update add ca-certificates ruby ruby-irb ruby-json ruby-rake \
-    ruby-bigdecimal ruby-io-console libstdc++ tzdata postgresql-client nodejs \
-    linux-headers libc-dev zlib libxml2 libxslt libffi less
+RUN apk update && apk --update add \
+    ca-certificates \
+    tzdata \
+    linux-headers \
+    libxml2 \
+    libxslt \
+    postgresql-client \
+    nodejs \
```

Comment:
Would certainly speed stuff up to ditch the asset pipeline.  But since this PR already does 2 things ... I'd vote for opening up a chore-ticket to come back to later.